includes/multicastAlarm.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+"""
+multicastAlarm is the function to enable BOSwatch to deal networks that optimise the transmission of POCSAG telegrams
+
+@author: Fabian Kessler
+
+@requires: Configuration has to be set in the config.ini
+"""
+
+import logging # Global logger
+import time    # timestamp for multicastAlarm
+
+from includes import globalVars  # Global variables
+
+multiList = []
+
+def newEntrymultiList(eatyp, eapoc_id, eapoc_sub, eapoc_text):
+	"""
+	add entry to multi alarm list and remove old entries
+
+	@return:    nothing
+	"""
+	global multiList
+	tmpmultiList = []
+	timestamp = int(time.time())
+	# multicastAlarm processing if enabled and delimiter RIC has been received
+	if eapoc_id == globalVars.config.get("multicastAlarm", "multicastAlarm_delimiter_ric"):
+		multiList = []
+		logging.debug("multicastAlarm delimiter RIC received --> buffer cleared  %s  %s %s ", eapoc_id, eapoc_sub, eapoc_text)
+	else:
+		multiList.append([eatyp, eapoc_id, eapoc_sub, eapoc_text.strip(), timestamp])
+		logging.debug("Added %s  %s %s to multiList", eapoc_id, eapoc_sub, eapoc_text)
+		# check for old entries in multiList
+		for i in range(len(multiList)):
```

Comment:
Den angemeckerten Vorschlag sollte man evtl noch ausbessern.
Die ganzen `Found indentation with tabs instead of spaces` sind zu ignorieren, da hatte ich in Codacy eine Falsche Funktion angeklickt, ausversehen.

Reply:
ist erledigt